### DISCUSSION TITLE
Tree Structure for MPI-INF-3DHP

### DISCUSSION THREAD

**Discussion by smahmood00** [open-mmlab/mmpose#1462] [Q&A]
Could you please tell me what should be the tree structure for MPI-INF-3DHP dataset in the mmpose "data" folder in order to do inference and training? There are various files in the dataset zip folder downloaded from the official site. Thanks in advance

**Comment by liqikai9** [comment]
You can refer to [this](https://github.com/open-mmlab/mmpose/blob/master/docs/en/tasks/3d_body_mesh.md#mpi-inf-3dhp).

**Reply by smahmood00** [reply]
but I want to infer 3D single view body pose estimation from image, not 3d human mesh recovery

**Comment by liqikai9** [comment]
You can refer to this [config](https://github.com/open-mmlab/mmpose/blob/master/configs/body/3d_kpt_sview_rgb_img/pose_lift/mpi_inf_3dhp/simplebaseline3d_mpi-inf-3dhp.py) and format the tree structure as the config specifies.

**Comment by ly015** [comment]
We provide a [script](https://github.com/open-mmlab/mmpose/blob/master/tools/dataset/preprocess_mpi_inf_3dhp.py) to convert raw downloaded MPI-INF-3DHP files into the desired structure. However the illustration of organized folder structure is missing in the docs. It will be added in the future. You may refer to the script to help prepare your data for now.
